python/agents/utils/utils.py

```
@@ -0,0 +1,59 @@
+            )
+
+        if k in ["volume", "liquidity"]:
+            description += f" This market has a current {k} of {v}."
+    print("\n\ndescription:", description)
+
+    market_object["description"] = description
+
+    return market_object
+
+
+def preprocess_local_json(file_path: str, preprocessor_function: function) -> None:
```

Ruff (0.11.9)

Undefined name `function`

(F821)